src/infra/api/tags/types.ts

```diff
@@ -14,6 +14,10 @@ export interface GetTagSearchResponse {
   tags: Pick<Tag, "tagId" | "name" | "viewCount">[];
 }
 
+export interface GetMemeTagsByIdResponse {
+  tags: Tag[];
+}
```

Comment:
response 타입이 `GetPopularTagResponse` 랑 같은 데 어차피 둘 다 tag 를 response 한다면 
```suggestion
export interface GetTagsResponse {
  tags: Tag[];
}
```
라고 공통으로 쓰는 건 어떻게 생각하시나요 👀

Reply:
음 각 api 마다 response가 나중에 어떻게 바뀔 지 몰라서 일단 분리 해놨어용~
`type GetMemeTagsByIdResponse = GetPopularTagsResponse` 같은 type alias는 어떻게 생각하시나요?

Reply:
type alias 로 바꿔도 좋은 것 같아요~!

Reply:
그냥 다른 API니까 따로 분리하는게 더 나을듯!